tests/server/api/v1/test_records.py

```diff
@@ -101,7 +101,7 @@ def create_multi_label_selection_questions(dataset: "Dataset") -> None:
         ),
     ],
 )
-def test_create_record_response(
+def test_create_record_response_with_valid_values(
```

Comment:
Regarding the new test name, we should parameterize also the `status` and create responses with partial values (this is a difference between `submitted` and `draft`)